src/components/QueryCondition.vue

```diff
@@ -32,6 +32,7 @@
 					:error="valueError"
 					:datatype="datatype"
 				/>
+				<SubclassCheckbox @subclass-check="setSubclasses" :isChecked="subclasses(conditionIndex)" />
```

Comment:
This should only be shown for properties with datatype `wikibase-item`

Reply:
Oh, I understood it was only checked when datatype is `wikibase-item`  but always shown? Will update.

Reply:
addressed!